projects/RabbitMQ.Client/client/impl/TcpClientAdapter.cs

```diff
@@ -27,7 +27,7 @@ public virtual async Task ConnectAsync(string host, int port)
                 throw new ArgumentException($"No ip address could be resolved for {host}");
             }
 
-            await ConnectAsync(ep, port);
+            await ConnectAsync(ep, port).ConfigureAwait(false);
```

Comment:
Is there any difference in using `ConfigureAwait(false)` at the end of the method? After all, after this line there is no code and there is no next step of the state machine. Or am I missing something?

Reply:
OK. Also I use ca2007 in everyday work. I have to always disable its warnings for test projects.